test/tools/jsontests/BlockChainTests.h

```diff
@@ -53,6 +53,13 @@ class BCGeneralStateTestsSuite : public BlockchainValidTestSuite
     boost::filesystem::path suiteFillerFolder() const override;
 };
 
+class BCGeneralStateTestsSuiteLegacyConstantinople : public BCGeneralStateTestsSuite
```

Comment:
Why not put this in a separate header for Legacy tests?

It's difficult now to find what is where: usual state tests are in StateTests.h + StateTests.cpp
usual blockchain tests are in BlockChainTests.h + BlockChainTests.cpp

and now legacy state tests are in StateTests.h + LegacyTestBoost.cpp
legacy blockchain tests in BlockChainTests.h + LegacyTestBoost.cpp

I'd create a header for LegacyBoostTests and also maybe split them into separate files for state and blockchain ones.